src/images/maps/geom_adm0.R

```diff
@@ -1,22 +1,37 @@
 box::use(gg = ggplot2)
+box::use(dplyr)
+box::use(purrr)
+box::use(sf)
+box::use(units)
 
 box::use(../../utils/get_iso3_sf)
 box::use(../../utils/iso3_shift_longitude)
 
+
 #' Geom for country boundaries
 #'
 #' Adds country boundaries for ISO3 code. Since `get_adm0_sf()` looks for custom
 #' basemaps, then OCHA CODs, then UN Geodata services by filtering, we throw
 #' an error if the returned data is `NULL` or a 0 row data frame.
 #'
 #' @param iso3 ISO3 code
+#' @param ... Additional sf class objects with geometry
 #'
 #' @returns Geom for the country boundaries
 #'
 #' @export
-geom_adm0 <- function(iso3) {
+geom_adm0 <- function(iso3, ...) {
   sf_adm0 <- get_iso3_sf$get_iso3_sf(iso3, "adm0")
 
+
+  additional_geoms <- list(...)
+  if (length(additional_geoms) > 0) {
+    purrr$map(
+      .x = additional_geoms,
+      .f = \(x) assert_within_distance(x, sf_adm0)
+    )
+  }
+
   if (is.null(sf_adm0) || nrow(sf_adm0) == 0) {
```

Comment:
You need to run your assertion after we check that `sf_adm0` isn't `NULL` or missing, because yours will fail in these cases.

Reply:
fixed  [6ebb2b7](https://github.com/OCHA-DAP/hdx-signals/pull/113/commits/6ebb2b7f07bcfc994dcf49c84457c407b097545b)